BUILD.md

```diff
@@ -222,6 +222,13 @@ Ideally, pull requests targeting performance improvements should include such a
 ## Code Coverage
 
 Code coverage of the Tosca project is a work in progress. As the different systems get integrated, more features will be added and documented here.
+Go has native support for coverage, which makes getting this metric for `lfvm` or `geth` driver runs quite simple.
+Code coverage for C++ this is not the case, hence a little more setup is needed for this task. 
```

Comment:
nit: I think this sentence has a grammar issue and an arguable style issue ...

The grammar issue: It should probably be "For code coverage of C++, this is not the case."

The style issue is something you may ignore; but indicating that it takes "a little more setup" would tell me that there is a solution and it takes the user more setup to get it going. Not that it takes us as developers more time to figure out how to do it.

Reply:
Your new version still implies that the solution is already there, but the BUILD files just does not to tell you how to do it. But never mind, this will be fixed once the C++ support becomes available.